pytradfri/color.py

```diff
@@ -86,82 +92,34 @@ def kelvin_to_xyY(T, white_spectrum_bulb=False):
     return {X: x, Y: y}
 
 
-def xyY_to_kelvin(x, y):
-    # This is an approximation, for information, see the source.
-    # Source: https://en.wikipedia.org/wiki/Color_temperature#Approximation
-    # Input range for x and y is 0-65535
-    n = (x/65535-0.3320) / (y/65535-0.1858)
-    kelvin = int((-449*n**3 + 3525*n**2 - 6823.3*n + 5520.33) + 0.5)
-    return kelvin
-
-
-def rgb2xyzA(r, g, b):
-    # Uses CIE standard illuminant A = 2856K
-    # src: http://www.brucelindbloom.com/index.html?Eqn_RGB_XYZ_Matrix.html
-    # calculation https://gist.github.com/r41d/43e14df2ccaeca56d32796efd6584b48
-    X = 0.76103282*r + 0.29537849*g + 0.04208869*b
-    Y = 0.39240755*r + 0.59075697*g + 0.01683548*b
-    Z = 0.03567341*r + 0.0984595*g + 0.22166709*b
-    return X, Y, Z
+# https://stackoverflow.com/questions/29643352/converting-hex-to-rgb-value-in-python
+def hex_to_rgb(hex):
```

Comment:
Do we still need these conversion methods?

Reply:
Hmm, not sure why you see this hex_to_rgb method? I don't have it in my local branch, which is in sync with the one on github.

Reply:
This isn't in an older commit, are you sure your local copy is in sync with GitHub?